Resources/Textures/_CM14/Structures/Walls/girder.rsi/meta.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from cmss13 at commit https://github.com/cmss13-devs/cmss13/commit/f74929c306a5d2ec5832abcdc695aea1b0786664",
```

Comment:
This should be a link to the actual file at the given commit, not just the commit, if you copy pasted the github .dmi link into the latest version of RSIEdit, it should do that for you automatically.

I can't actually find the girder in that commit though, are you sure it comes from there?